spec/features/bulk_discounts/index_spec.rb

```diff
@@ -0,0 +1,67 @@
+    @invoice_2 = Invoice.create!(customer_id: @customer_1.id, status: 2)
+    @invoice_3 = Invoice.create!(customer_id: @customer_2.id, status: 2)
+    @invoice_4 = Invoice.create!(customer_id: @customer_3.id, status: 2)
+    @invoice_5 = Invoice.create!(customer_id: @customer_4.id, status: 2)
+    @invoice_6 = Invoice.create!(customer_id: @customer_5.id, status: 2)
+    @invoice_7 = Invoice.create!(customer_id: @customer_6.id, status: 1)
+
+    @item_1 = Item.create!(name: "Shampoo", description: "This washes your hair", unit_price: 10, merchant_id: @merchant1.id)
+    @item_2 = Item.create!(name: "Conditioner", description: "This makes your hair shiny", unit_price: 8, merchant_id: @merchant1.id)
+    @item_3 = Item.create!(name: "Brush", description: "This takes out tangles", unit_price: 5, merchant_id: @merchant1.id)
+    @item_4 = Item.create!(name: "Hair tie", description: "This holds up your hair", unit_price: 1, merchant_id: @merchant1.id)
+
+    @ii_1 = InvoiceItem.create!(invoice_id: @invoice_1.id, item_id: @item_1.id, quantity: 1, unit_price: 10, status: 0)
+    @ii_2 = InvoiceItem.create!(invoice_id: @invoice_1.id, item_id: @item_2.id, quantity: 1, unit_price: 8, status: 0)
+    @ii_3 = InvoiceItem.create!(invoice_id: @invoice_2.id, item_id: @item_3.id, quantity: 1, unit_price: 5, status: 2)
+    @ii_4 = InvoiceItem.create!(invoice_id: @invoice_3.id, item_id: @item_4.id, quantity: 1, unit_price: 5, status: 1)
+    @ii_5 = InvoiceItem.create!(invoice_id: @invoice_4.id, item_id: @item_4.id, quantity: 1, unit_price: 5, status: 1)
+    @ii_6 = InvoiceItem.create!(invoice_id: @invoice_5.id, item_id: @item_4.id, quantity: 1, unit_price: 5, status: 1)
+    @ii_7 = InvoiceItem.create!(invoice_id: @invoice_6.id, item_id: @item_4.id, quantity: 1, unit_price: 5, status: 1)
+
+    @transaction1 = Transaction.create!(credit_card_number: 203942, result: 1, invoice_id: @invoice_1.id)
+    @transaction2 = Transaction.create!(credit_card_number: 230948, result: 1, invoice_id: @invoice_3.id)
+    @transaction3 = Transaction.create!(credit_card_number: 234092, result: 1, invoice_id: @invoice_4.id)
+    @transaction4 = Transaction.create!(credit_card_number: 230429, result: 1, invoice_id: @invoice_5.id)
+    @transaction5 = Transaction.create!(credit_card_number: 102938, result: 1, invoice_id: @invoice_6.id)
+    @transaction6 = Transaction.create!(credit_card_number: 879799, result: 1, invoice_id: @invoice_7.id)
+    @transaction7 = Transaction.create!(credit_card_number: 203942, result: 1, invoice_id: @invoice_2.id)
+
+    @bulk_discount_1 = BulkDiscount.create!(percentage_discount: 10, quantity_threshold: 20.0, merchant_id: @merchant1.id)
+    @bulk_discount_2 = BulkDiscount.create!(percentage_discount: 15, quantity_threshold: 30.0, merchant_id: @merchant1.id)
+    @bulk_discount_3 = BulkDiscount.create!(percentage_discount: 20, quantity_threshold: 40.0, merchant_id: @merchant1.id)
+
+    visit "/merchants/#{@merchant1.id}/bulk_discounts"
```

Comment:
Having the routing in the before(:each) kind of helps me make more sense as to when and where I need to stop and start testing for a user story. So instead of thinking "where do I start this next part in this test?" I can consider that if it's not on the visit page in the before block, then the testing should be in another test file.